client/my-sites/stats/activity-log/index.jsx

```diff
@@ -510,6 +511,7 @@ class ActivityLog extends Component {
 				<StatsFirstView />
 				<SidebarNavigation />
 				<StatsNavigation selectedItem={ 'activity' } siteId={ siteId } slug={ slug } />
+				{ siteId && <ActivityLogUpgradeNotice siteId={ siteId } /> }
```

Comment:
note that here we could conditionally render this based on the `/rewind` info

```js
{ siteId && rewindState.reason === 'missing_plan' && <ActivityLog…/> }
```

Reply:
actually at that point I don't think we need the `ActivityLogUpgradeNotice` and instead could just inline the banner

```js
{ siteId && rewindState.reason === 'missing_plan' && (
  <Banner
    plan="personal-bundle"
    href=…
    …
  />
}
```